openedx/features/smart_referral/helpers.py

```diff
@@ -57,3 +59,23 @@ def get_email_domain(email):
     Return email domain name e.g "google.com", "yahoo.com" and "outlook.com" etc by splitting email after "@".
     """
     return email.split('@')[1] if email else None
+
+
+def filter_referred_contacts(contacts, user):
```

Comment:
In the AC it says 

- Filter out contact if it has already registered on our platform

I can't seem to find that filter here. How we are handling that logic?

Reply:
We are already showing only non-platform contact on the widget. So, I have added my filter on **non-platform contacts** only. See this [line](https://github.com/philanthropy-u/edx-platform/pull/1644/files#diff-18e5c3d4ca9be3c6408a21ce14e264c8R40)